src/scss/_guidance.scss

```diff
@@ -0,0 +1,77 @@
+/// @access private
+/// Outputs styling for accessibility components
+
+@mixin _oVideoGuidance {
+    $white: oColorsGetPaletteColor('white');
+
+    .o-video__guidance {
+        @include oTypographySans(-1);
+        background-color:  oColorsMix($color: 'black', $background: 'transparent', $percentage: 75);
+        padding: 10px 16px;
+        -webkit-font-smoothing: antialiased;
```

Comment:
```suggestion
        // sass-lint:disable no-vendor-prefixes
        -webkit-font-smoothing: antialiased;
        // sass-lint:enable no-vendor-prefixes
```

Reply:
🤦‍♂ Had this in a rebased commit I hadn't to force pushed